src/app/pages/MediaArticlePage/PagePromoSections/LastestMediaSection/index.styles.tsx

```diff
@@ -0,0 +1,49 @@
+import pixelsToRem from '#app/utilities/pixelsToRem';
+import { Theme, css } from '@emotion/react';
+
+const styles = {
+  LatestMediaSection: ({ mq, isDarkUi, palette }: Theme) =>
```

Comment:
**nitpick**: Should the style names start with lowercase - given that they are just fields on an object?
```suggestion
  latestMediaSection: ({ mq, isDarkUi, palette }: Theme) =>
```